ocf_data_sampler/load/site.py

```diff
@@ -0,0 +1,26 @@
+import pandas as pd
+import xarray as xr
+
+from ocf_data_sampler.config.model import Site
+
+
+def open_site(sites_config: Site) -> xr.DataArray:
+
+    # Load site generation xr.Dataset
+    data_ds = xr.open_dataset(sites_config.filename)
+
+    # Load site generation data
+    metadata_df = pd.read_csv(sites_config.metadata_filename)
+
+    metadata_df.set_index("site_id", inplace=True, drop=True)
+
+    # Add coordinates
+    ds = data_ds.assign_coords(
+        latitude=(metadata_df.latitude.to_xarray()),
+        longitude=(metadata_df.longitude.to_xarray()),
+        capacity_kwp=data_ds.capacity_kwp,
+    )
+
+    return ds.generation_kw
```

Comment:
Do we still want any of [these](https://github.com/openclimatefix/ocf_datapipes/blob/main/ocf_datapipes/load/pv/pv.py#L197) checks included? Some of them might still be useful

Reply:
Ive added a few in, they are slightly different, but still worth doing